src/databaseTests/build.gradle

```diff
@@ -19,8 +19,8 @@ dependencies {
 }
 
 test {
-    dependsOn 'copySpec', ':startDatabase', 'createTemplateFromDatabase'
```

Comment:
These tasks will fail on Travis because they require the local registry, so have stopped them being triggered automatically and now manually executing them before/after tests run on TC https://github.com/vimc/montagu-api/pull/414/files#diff-3e0e48b56ab9b06df6f4e2465783c472R13